apps/src/gamelab/GameLabSprite.js

```diff
@@ -213,7 +204,7 @@ module.exports.createSprite = function (x, y, width, height) {
    * @method
    */
   s.isTouching = function (target) {
-    return this.createGroupState('collide', target, undefined, false);
+    return this.createGroupState('overlap', target);
```

Comment:
It should.  There used to be some subtle differences between the `overlap` and `collide` code and `collide` was more accurate, so we implemented `isTouching` as `collide` with the `modifyPosition` flag as the fourth argument - not great.  Now those two are using the same underlying implementation so we don't need the workaround.

One subtle difference - we set the `sprite.touching.top|left|right|bottom` properties where we didn't before - but we don't expose the affected parts in our toolbox or curriculum and I think for that behavior change it's more predictable now.